<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>go2IDLE</class>
 <widget class="QMainWindow" name="go2IDLE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>go2IDLE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>135</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_packets_transferred">
       <property name="text">
        <string># Packets Transferred:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_acks">
       <property name="text">
        <string># ACKs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_naks">
       <property name="text">
        <string># NAKs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_bit_error_rate">
       <property name="text">
        <string>Bit Error Rate:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>97</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="stop_button">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="menu_select_file"/>
    <addaction name="menu_settings"/>
    <addaction name="menu_disconnect"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_select_file">
   <property name="text">
    <string>Select File</string>
   </property>
  </action>
  <action name="menu_disconnect">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menu_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>menu_disconnect</sender>
   <signal>triggered()</signal>
   <receiver>go2IDLE</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
